graphql/resolvers/user.js

```diff
@@ -1,10 +1,24 @@
-import * as dbUsers from '../../dataLayer/mongo/user';
+import debug from 'debug';
+import {
+  getUser,
+  createUser,
+  updateChallengeMap
+} from '../../dataLayer/mongo/user';
+
+const log = debug('fcc:resolvers:user');
 
 export const userResolvers = {
   Query: {
-    users: (_, args) => dbUsers.getUsers(args)
+    // TODO(Bouncey): requires auth protection
```

Comment:
See comments about TODO earlier on. You could probably convince me this is ok for now, as things are in a state of flux while we get this project production ready. However, you would be giving a good example for new contributors, and it will be easier for the whole team to see what's going on by using issues for this.

Reply:
I agree that it would be really good to have this in place now. If auth had been implemented at the project level, it would definitely be implemented for this query and the user related mutations.

In my opinion it should be a separate PR that implements some form of auth handling, and that there is no place for it in this PR. I am looking in to this on a separate branch.